### DISCUSSION TITLE
WeatherKit无法使用

### DISCUSSION THREAD

**Discussion by xiaoxiao8803** [NSRingo/.github#15] [Q&A 问答] [answered]
ios18.1 iphone13 stash采用默认设置 求解
![图片_20241114150214](https://github.com/user-attachments/assets/78f5eeb7-0348-4e7a-8939-4f029fb295bd)
![图片_20241114150207](https://github.com/user-attachments/assets/0d97c061-2e0c-4d2d-9e18-f58ee653d2fd)

**Comment by VirgilClyne** [comment, marked as answer]
Stash 的基础使用教程参见:
https://stash.wiki/
